doc/source/whatsnew/v0.13.0.rst

```diff
@@ -153,12 +153,12 @@ API changes
 
   Added the ``.bool()`` method to ``NDFrame`` objects to facilitate evaluating of single-element boolean Series:
 
-  .. ipython:: python
+  .. code-block:: python
 
-     pd.Series([True]).bool()
-     pd.Series([False]).bool()
-     pd.DataFrame([[True]]).bool()
-     pd.DataFrame([[False]]).bool()
+     >>> pd.Series([True]).bool()
+     >>> pd.Series([False]).bool()
+     >>> pd.DataFrame([[True]]).bool()
+     >>> pd.DataFrame([[False]]).bool()
```

Comment:
Need to also include the output from these lines here.

Reply:
Can you check it